bedrock/products/models.py

```diff
@@ -109,3 +111,205 @@ def get_context(self, request, *args, **kwargs):
+                ),
+                FieldPanel("split_image", heading="Intro Image"),
+            ],
+            heading="Intro Block",
+        ),
+    ]
+
+    class Meta(TranslatableMixin.Meta):
+        verbose_name = "Monitor Call To Action Snippet"
+        verbose_name_plural = "Monitor Call To Action Snippets"
+
+    def __str__(self):
+        return f"{self.split_heading} - CTA Snippet"
+
+
+class MonitorArticleIndexPage(AbstractBedrockCMSPage):
+    split_heading = models.CharField(
+        max_length=60,
+        blank=False,
+    )
+    split_subheading = models.CharField(
+        max_length=60,
+        blank=True,
+    )
+    split_content = RichTextField(null=True, blank=True, features=["bold", "italic", "link", "ul", "ol"])
+    split_button_text = models.CharField(
+        verbose_name="Link Text",
+        max_length=40,
+        blank=False,
+    )
+    split_button_link = models.URLField(
+        verbose_name="Link URL",
+        blank=False,
+    )
+    split_image = models.ForeignKey(
+        "cms.BedrockImage",
+        null=True,
+        blank=False,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    articles_heading = models.CharField(
+        max_length=100,
+        blank=False,
+    )
+    articles_subheading = models.CharField(
+        max_length=255,
+        blank=True,
+    )
+    call_to_action_bottom = models.ForeignKey(
+        "products.MonitorCallToActionSnippet",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    content_panels = AbstractBedrockCMSPage.content_panels + [
+        MultiFieldPanel(
+            [
+                FieldPanel(
+                    "split_heading",
+                    heading="Intro Heading",
+                ),
+                FieldPanel(
+                    "split_subheading",
+                    heading="Intro Subheading (optional)",
+                ),
+                FieldPanel(
+                    "split_content",
+                    heading="Intro Content (optional)",
+                ),
+                FieldRowPanel(
+                    [
+                        FieldPanel("split_button_text", heading="Intro Button text"),
+                        FieldPanel("split_button_link", heading="Intro Button link"),
+                    ]
+                ),
+                FieldPanel("split_image", heading="Intro Image"),
+            ],
+            heading="Intro Split Block",
+        ),
+        MultiFieldPanel(
+            [
+                FieldPanel("articles_heading"),
+                FieldPanel("articles_subheading"),
+            ],
+            heading="Article Section Heading",
+        ),
+        HelpPanel(content="The articles will be linked to here"),
+        FieldPanel("call_to_action_bottom"),
+    ]
+
+    template = "products/monitor/cms/index.html"
+    AbstractBedrockCMSPage._meta.get_field("title").help_text = "Heading (use sentence case)"
```

Comment:
@stephaniehobson Ah, I see why we're having strange things happen. 

I didn't realise that we can't do this hacking of the `title` field, unfortunately - nor the other two tweaks of the `title` field at the bottom of the file, because they'll actually change the spec of the `title` field on the Wagtail-owned `Page` model, not on our Bedrock model classes, which in turn generates that `wagtailcore_0095` migration that we 
a) don't commit the the source repo 
and 
b) can't commit because it will inevitably clash in the future with Wagtails actual 0095 migration when it exists.

So, worst case we have to live with slightly less helpful/steerting labelling for the `title` field, I'm afraid

Reply:
oooooooh.